.github/workflows/elixir.yml

```
@@ -32,7 +32,7 @@ jobs:
         key: ${{ runner.os }}-build-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
         restore-keys: |
           ${{ runner.os }}-build-
-    - uses: actions/setup-elixir@v1
+    - uses: erlef/setup-beam@v1
```

If we want to use dependabot for actions, we can also start to use the fully qualified version number:

```suggestion
    - uses: erlef/setup-beam@v1.7.0
```